From 8a29f4a8fb80a9545518a3930704d3f05c3064f1 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 20 Apr 2020 13:41:10 +0100
Subject: [PATCH] Revert "spi: spidev: Fix CS polarity if GPIO descriptors are
 used"

This reverts commit 83b2a8fe43bda0c11981ad6afa5dd0104d78be28.
---
 drivers/spi/spidev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -428,7 +428,7 @@ spidev_ioctl(struct file *filp, unsigned
 			}
 
 			if (ctlr->use_gpio_descriptors && spi_get_csgpiod(spi, 0))
-				tmp |= SPI_CS_HIGH;
+				{ /*tmp |= SPI_CS_HIGH;*/ }
 
 			tmp |= spi->mode & ~SPI_MODE_MASK;
 			spi->mode = tmp & SPI_MODE_USER_MASK;
